Makefile

```diff
@@ -13,6 +13,11 @@ CPU_PREFIX_310 := $(REGISTRY_REPO):py-3.10-
 CUDA_113_PREFIX := $(REGISTRY_REPO):cuda-11.3-
 CUDA_118_PREFIX := $(REGISTRY_REPO):cuda-11.8-
 ROCM_56_PREFIX := $(REGISTRY_REPO):rocm-5.6-
+ROCM_57_PREFIX := $(REGISTRY_REPO):rocm-5.7-
+ROCM_60_PREFIX := $(REGISTRY_REPO):rocm-6.0-
+ROCM_61_PREFIX := $(REGISTRY_REPO):rocm-6.1-
+ROCM_60_TF_PREFIX := tensorflow-infinity-hub:tensorflow-infinity-hub
```

Comment:
Why other images are stored in `REGISTRY_REPO := environments`, or a repo with a `-dev` suffix, but this one is not?